frontend/public/locales/en/translation.json

```diff
@@ -458,15 +458,15 @@
     "institutions_democracy": "Topics related to the organization of political life, democracy, territories and collectivities, public service and overseas",
     "labour_economy": "Topics related to jobs, unemployment, purchasing power, taxation, public deficits and debt",
     "solidarity": "Topics related to societal issues, inequality and discrimination, disability, violence against women and immigration",
-    "pedagogy": "'Clear & pedagogical' is a measure of how clear and educational the content is. The more the content can guide the spectators in their understanding, the higher this criterion will be rated",
-    "importance": "'Important & actionable' aims to highlight content with information that does not have the attention it should have for the good of humanity and beyond and that allows to act concretely in an impactful way",
-    "layman_friendly": "'Layman-friendly' aims to identify contents that would be suitable to those with very limited prior knowledge",
-    "entertaining_relaxing": "'Entertaining & relaxing' evaluates videos based on whether it is relaxed and entertaining for viewers",
-    "engaging": "'Engaging & thought-provoking' evaluates videos based on the viewer's desire to go further after watching and/or the reconsideration of their beliefs",
-    "diversity_inclusion": "'Diversity & inclusion' aims to identify content that promotes a more diverse and inclusive view of the great diversity of humanity's ethnicities, beliefs and preferences",
-    "better_habits": "'Encourages better habits' aims to highlight content that encourages and motivates viewers to improve themselves by adopting better and healthier habits for themselves, for society or for the planet",
-    "backfire_risk": "'Resilience to backfiring risks' aims to measure the probability that a content will create a misunderstanding with the viewer: the higher the risk that the user misunderstands or reacts negatively to the information presented, the higher this criterion will be rated",
-    "reliability": "'Reliable & not misleading' aims to measure the trustworthiness of the information presented in a content, but also the extent to which the information may be misleading"
+    "reliability": "Is the presented information trustworthy, robustly backed and properly nuanced?",
+    "pedagogy": "How efficiently does the content guide viewers in their understanding?",
+    "importance": "Can the focus on this topic have a significantly positive impact on the world?",
```

Comment:
Je propose de modifier:
"Can the focus..."
en
"Can additional focus..."
(ou éventuellement "Can paying more attention...")
=> Pour le cas où un sujet est important, mais qu'on en parler déjà assez voire trop

Reply:
Good for me.